src/Selenium2Library/keywords/_tableelement.py

```diff
@@ -73,6 +73,37 @@ def table_cell_should_contain(self, table_locator, row, column, expected, loglev
             self.log_source(loglevel)
             raise AssertionError(message)
 
+    def table_cell_should_not_contain(self, table_locator, row, column, expected, loglevel='INFO'):
```

Comment:
For all the new code, try to follow pep8 maximum line length: https://www.python.org/dev/peps/pep-0008/#maximum-line-length. Fix in all places where it is exceeded in code which you have written. You do not need fix the existing code.